elliottlib/errata.py

```diff
@@ -757,6 +757,11 @@ def get_advisory(advisory_id):
     return ErrataConnector()._get(f'/api/v1/erratum/{advisory_id}')
 
 
+def get_advisory_greenwave(advisory_id):
+    result = ErrataConnector()._get(f'/api/v1/external_tests?filter[test_type]=greenwave_cvp&filter[status]=FAILED&filter[active]=true&filter[errata_id]={advisory_id}')
```

Comment:
updated with page[size] filter, Although I think this method is only used for conditional judgment.

Reply:
You can't assume a certain use case once this function is introduced into the library, though it might be sufficient for your current use case. Please make this API call correct before getting it into the library.

Reply:
Here's an example showing how to handle pagination: https://github.com/openshift/elliott/blob/d264ff7022d9c2a1e54d28fa4345873410b295a6/elliottlib/errata.py#L641

Reply:
Also the function name is too generic. It sounds like a function to get all greenwave tests, but the reality is to get failed CVP tests.